BossMod/Replay/ReplayManagementWindow.cs

```diff
@@ -171,6 +171,6 @@ private unsafe string GetPrefix()
         if (cf->IsSilenceEcho)
             prefix += "_NE";
 
-        return prefix;
+        return Utils.CleanFilename(prefix);
```

Comment:
Uhh why exactly is it needed? There's a call to StringToIdentifier above already, which removes all non-alphanumeric characters - what exactly is the part of the name that creates invalid characters?

Reply:
my fault. This branch was from before the last changes to auto record but I didn't PR it then cause I couldn't test it. Tested it now then remerged master but didn't see your changes already fixed this too